php/libraries/Module.class.inc

```diff
@@ -150,7 +154,7 @@ class Module extends \LORIS\Router\PrefixRouter implements RequestHandlerInterfa
      * Returns true if this module should be publicly available to people who
      * are not logged in to LORIS.
      *
-     * @return bool (false: not that this is not currently possible because main.php
+     * @return bool (false: note that this is not currently possible because main.php
```

Comment:
I know you're just fixing a typo, but I don't think this note is relevant anymore (main.php is only used for instrument loading on major right now.)

Reply:
Should I delete it?

Reply:
Might as well since you're changing it anyways, otherwise we'll have to do it later